src/ralph/assets/migrations/0002_auto_20150519_0652.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('assets', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='baseobject',
+            name='parent',
+            field=models.ForeignKey(blank=True, to='assets.BaseObject', null=True),
```

Comment:
it's already merged: https://github.com/allegro/ralph/blob/ng/src/ralph/assets/migrations/0002_auto_20150520_0619.py